c_wrapper/lua-mongoc-collection.c

```diff
@@ -650,53 +650,54 @@ lua_mongo_collection_create_index(lua_State *L)
     int keys_index = 3;
     int options_index = 4;
 
-    if (!(lua_istable(L, keys_index))) {
+    if(!(lua_istable(L, keys_index))){
         luaL_error(L, "keys parameter must be a table");
-    }
-
-    if (!(lua_isnil(L, keys_index))){
+    }else{
```

Comment:
this else case isn't necessary if this line is true https://github.com/mongodb-labs/mongorover/pull/74/files#diff-b3084e8d014c2b0adf0231692fda4e23R675

This else should be the else case for the `if(lua_isstring(L, -9)){`